src/utils.ts

```diff
@@ -53,4 +53,61 @@ export const formatDate = () => {
         + appendLeadingZeroes(currentDateTime.getMinutes()) + ":"
         + appendLeadingZeroes(currentDateTime.getSeconds())
     return formattedDate;
+}
+
+// ** Product component **
+
+export const removeSymbols = (sentence: string): string => {
+    const regex: RegExp = /\$|\%|\-/g;
+    return sentence.replace(regex, '');
+}
+
+export const checkProductInformation = (productInformation: any): ProductFormErrors => {
```

Comment:
this is almost Awesome except productInformation: any) :D no any please we should know what we expect